mix implementation and integration in js (#340)
waku-org/pm: FURPS/core/mix.md

1. `wakunode2` for intermediary and exit nodes.
2. nwaku CLI for sender nodes.

> 3. Browser based apps built using js-waku should support acting as entry nodes.

Reviewer: Perhaps just

```suggestion
3. Browser-based apps on js-waku as entry nodes
```

to match the other items' phrasing?
